drf_braces/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 __author__ = 'Miroslav Shubernetskiy'
 __email__ = 'miroslav@miki725.com'
-__version__ = '0.2.3'
+__version__ = '0.3.0'
```

Comment:
I understand the general purpose for swappableserializer. Do we have any use cases in DT that we need this?